ui/page/governance/agenda_vote_modal.go

```diff
@@ -0,0 +1,374 @@
+package governance
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"gioui.org/font/gofont"
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/widget"
+	"gioui.org/widget/material"
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/page/staking"
```

Comment:
This seems to be where the import cycle issue is coming from. To use the VSP selector, you could simply move it to the components.go file.